docs/unified-test-documentation/dasharo-compatibility/100-coreboot-base-port.md

```diff
@@ -20,7 +18,7 @@ its main task is to set up everything for a C-environment.
 
 **Test setup**
 
-1. Proceed with the [Common](#common) section.
+1. Proceed with the [Common](#test-cases-common-documentation) section.
```

Comment:
If we remove `Common` section then we shouldn't change this reference to `Test cases common documentation`? I understand that I have to proceed with the Common section, which is removed (Of course it moves to the section correctly).

Reply:
improved in all files